controllers/components/workbenches/workbenches_controller.go

```diff
@@ -62,7 +61,6 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 		WithAction(initialize).
 		WithAction(devFlags).
 		WithAction(configureDependencies).
-		WithAction(security.NewUpdatePodSecurityRoleBindingAction(serviceAccounts)).
```

Comment:
I'm not following this in the entirety, but for Notebooks we really don't need to have anyuid scc, certainly not after https://issues.redhat.com/browse/RHOAIENG-72 got in, and IMO even before that we were good with any uid.

cc @opendatahub-io/notebook-devs for heads-up!

* https://github.com/opendatahub-io/kubeflow/pull/348